Marlin/ultralcd.cpp

```diff
@@ -1297,9 +1296,7 @@ void kill_screen(const char* lcd_msg) {
           LCDVIEW_CALL_NO_REDRAW
         #endif
       ;
-      if (no_reentrance) return;
       lcd_wait_for_homing();
```

Comment:
So, reverted my changes back. I guess the only difference is that Marlin here uses
```
if (axis_homed[X_AXIS] && axis_homed[Y_AXIS] && axis_homed[Z_AXIS])
  lcd_goto_screen(_lcd_level_bed_homing_done);
```
instead, which can easily done. I think it will be even better in order to stay in sync with them.

So, now I have working both manual and auto leveling. Just check my previous 2 commits where I tried both configurations. Works for me on A6.